apps/src/blockly/addons/functionEditor.js

```diff
@@ -356,4 +355,41 @@ export default class FunctionEditor {
       },
     };
   }
+
+  // Copy all procedure models from the hidden definition workspace to the editor workspace,
+  // if they are not already present in the editor workspace.
+  // This is needed when the hidden definition workspace is initialized with at least one
+  // procedure, so that the editor workspace knows about those procedures.
+  setUpEditorWorkspaceProcedures() {
+    Blockly.Events.disable();
+    const procedureDefinitions = this.editorWorkspace
```

Comment:
Could we rename this constant to `editorWorkspaceProcedureDefinitions` to help distinguish from procedures in `hiddenDefinitionWorkspace` for readability?